apps/etterlatte-saksbehandling-ui/client/src/components/behandling/sidemeny/SettPaaVent.tsx

```diff
@@ -50,6 +70,26 @@ export const SettPaaVent = ({ oppgave, redigerbar }: Props) => {
       {settPaaVent && (
         <>
           <hr />
+          <Text>Årsak til sett på vent</Text>
+          <Select
```

Comment:
Denne vises nå også når man tar noe av vent. Ser ikke helt bra ut

Reply:
Ikkesant, fordi den brukes på for å sette på og ta av ja.